PWGGA/GammaConvBase/AliV0ReaderV1.cxx

```diff
@@ -121,6 +121,7 @@ AliV0ReaderV1::AliV0ReaderV1(const char *name) : AliAnalysisTaskSE(name),
   fProduceImpactParamHistograms(kFALSE),
   fCurrentInvMassPair(0),
   fImprovedPsiPair(3),
+  fSDDSSDClusters(-1),
```

Comment:
this will throw a warning (reorder), please fix

Reply:
done